examples/flutter_gallery/lib/demo/animation/home.dart

```diff
@@ -292,17 +295,12 @@ class _AllSectionsView extends AnimatedWidget {
     // The layout's progress from from a column to a row. Its value is
     // 0.0 when size.height equals the maxHeight, 1.0 when the size.height
     // equals the midHeight.
-    final double tColumnToRow =
-      1.0 - ((size.height - midHeight) /
-             (maxHeight - midHeight)).clamp(0.0, 1.0);
-
+    final double tColumnToRow = 1.0 - ((size.height - midHeight) / (maxHeight - midHeight)).clamp(0.0, 1.0);
```

Comment:
i can live this this (the original isn't that much better), but this is definitely encroaching on the sketchy area where the formatting of an expression can dramatically affect the readability.

Reply:
(this might be an area where we need some sort of indicator to say "don't reformat this line". for example, having a trailing `//` at the end of the last line of an expression, meaning "don't touch anything in this expression".)

Reply:
if you add a comment at the end of the line you can force a linebreak. doubtful that formatting is better as you won't get the indent which requires semantic knowledge